### PR TITLE
20521 Fixed restoration flow when FF blocks entity type

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#20521

*Description of changes:*
- added LD id (Dev) to sample env file
- app version = 5.4.4
- updated browsers list
- fixed/hid some lint warnings
- fixed restoration/reinstatement flow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -24,9 +24,8 @@ VUE_APP_REGISTRIES_SEARCH_API_URL="https://bcregistry-dev.apigee.net/registry-se
 VUE_APP_REGISTRIES_SEARCH_API_VERSION="/api/v1"
 VUE_APP_REGISTRIES_SEARCH_API_KEY=
 
-
 #vaults launchdarkly
-VUE_APP_NAMEREQUEST_LD_CLIENT_ID=""
+VUE_APP_NAMEREQUEST_LD_CLIENT_ID="5fbec4b5d998e30990020007"
```
Comment (severinbeauvais) [inline .env.example:28]: This isn't really a "secret" (other UIs have this in their repo), and it's handy for devs starting work in this repo that need to copy `.env.example` to `.env`.

File: src/components/dialogs/confirm-name-request.vue [hunk 2]
```diff
@@ -181,8 +181,7 @@ export default class ConfirmNrDialog extends Mixins(
     staffPaymentComponent: StaffPayment
   }
 
-  @Prop({ default: async () => {} })
-  readonly onCancel: Function
+  @Prop({ default: async () => {} }) readonly onCancel: () => void
```
Comment (severinbeauvais) [inline src/components/dialogs/confirm-name-request.vue:184]: Lint fix.

File: src/components/new-request/search.vue [hunk 3]
```diff
@@ -242,46 +242,44 @@
 
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
-        <BusinessLookupFetch ref="MyBusinessLookup" />
```
Comment (severinbeauvais) [inline src/components/new-request/search.vue:245]: Ref wasn't used anywhere.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-756-shnluqg3.web.app

File: src/components/new-request/search.vue [hunk 4]
```diff
@@ -242,46 +242,44 @@
...
+        <BusinessLookupFetch />
         <SocietiesInfo
           v-if="isSocietyDisabled && showSocietiesInfo"
           :type="'restore'"
           :showDialog="showSocietiesInfo"
         />
-        <template v-else>
```
Comment (severinbeauvais) [inline src/components/new-request/search.vue:251]: Go to ⚙️ button above and hide whitespace changes.

All I did here was remove this code from the "else" block. This was a logic error. I tested thoroughly -- see ticket for screenshots.

File: src/components/new-request/search.vue [hunk 5]
```diff
@@ -242,46 +242,44 @@
...
-          <CompanyType v-if="getSearchBusiness && isBcRestorable && isSupportedRestoration(getEntityTypeCd)" />
-          <Jurisdiction
-            v-if="isSelectedXproAndRestorable"
+        <CompanyType v-if="getSearchBusiness && isBcRestorable" />
```
Comment (severinbeauvais) [inline src/components/new-request/search.vue:251]: The other part of the fix was to remove "is supported restoration" from this component. That logic is used elsewhere (for determining what buttons to show).

**Review by jamespaologarcia — APPROVED** [review]
LGTM

**Review by eve-git — APPROVED** [review]

**Pull request merged**